src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -223,14 +223,15 @@ public void storeLocation(Location location)
         PreparedQuery allLocationresults = datastore.prepare(allLocations);
         for (Entity entity : allLocationresults.asIterable()) {
           try {
-            double longitude = Double.parseDouble(entity.getProperty("longitude").toString());
-            double latitude = Double.parseDouble(entity.getProperty("latitude").toString());
-            if(Math.abs(locations.get(0).getLatitude()-latitude)<3 && Math.abs(locations.get(0).getLongitude()-longitude)<3)
+            double otherLongitude = Double.parseDouble(entity.getProperty("longitude").toString());
+            double otherLatitude = Double.parseDouble(entity.getProperty("latitude").toString());
+            if(Math.abs(locations.get(0).getLatitude()-otherLatitude)<3 && Math.abs(locations.get(0).getLongitude()-otherLongitude)<3)
```

Comment:
definition of near the the max difference of 3 in each of the longitude and latitude of charity locations